addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -233,6 +271,35 @@ var Colorpicker = Dialog.extend({
         this._updateHsl(hue, this.colorComponents.saturation, this.colorComponents.lightness);
         this._updateUI();
     },
+    /**
+     * Updates opacity when user starts clicking on opacity slider.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onMouseDownOpacitySlider: function (ev) {
+        this.sliderOpacityFlag = true;
+        this._onMouseMoveOpacitySlider(ev);
+    },
+    /**
+     * Updates opacity value on mouse drag over opacity slider.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onMouseMoveOpacitySlider: function (ev) {
+        if (!this.sliderOpacityFlag) {
+            return;
+        }
+
+        var z = ev.pageY - this.$colorOpacity.offset().top;
+        
+        if (z >= -1 && z <= 201){
+            var z = (1 - z/200);
```

Comment:
I do not understand where all your `200` values are coming from, is not this supposed to be the slider's height?